app/presenters/hyku/manifest_enabled_work_show_presenter.rb

```diff
@@ -58,5 +63,30 @@ def query_for_rendering(file_set_id)
                                         fl: [ActiveFedora.id_field, 'label_ssi', 'mime_type_ssi'],
                                         rows: 1)
       end
+
+      # Retrieve the required fields from the Form class. Derive the Form class name from the Model name.
+      #   The class should exist, but it may not if different amespaces are being used.
+      #   If it does not exist, rescue and return an empty Array.
+      #
+      # @return [Array] required fields
+      def metadata_fields
+        ns = "Hyrax"
+        model = solr_document['has_model_ssim'].first
+
+        "#{ns}::#{model}Form".constantize.required_fields
+      rescue
+        []
+      end
+
+      # Get the metadata value; return a String for single values and an Array of hashes for multiple values.
+      #
+      # @return [String or Array] field value(s)
+      def get_metadata_value(field)
+        if send(field).count == 1
+          send(field).first
+        else
+          send(field).collect { |f| { "@value" => f } }
```

Comment:
I'm not finding an example of this exact construct in the presentation API.  The examples I see of an array of hashes is for associating values with language codes.  There is one example hidden in the spec of an array of strings which might be a better fit here:
```
{
  "label": "Notes",
  "value": ["Text of note 1", "Text of note 2"]
}
```
That being said I tried to see if an array of hashes without language codes was valid using the online validator and it looks like it isn't really checking the contents of a metadata hash's value if it is an array.  But it does look like it is okay with hashes that only have "@value" and no "@language" in general.
https://github.com/iiif-prezi/iiif-prezi/blob/master/iiif_prezi/loader.py#L161-L190

Reply:
Thanks @cjcolvar - I just went with @value without @language - it rendered in UV fine. But I think your suggestion is a little cleaner.  Update incoming ...

Reply:
👍 How it looks in UV is probably the best test since I'm just looking at the json.